### DISCUSSION TITLE
Dolby Vision lost when converting from Profile 7 to Profile 8.1? [macOS]

### DISCUSSION THREAD

**Discussion by Shikariiii** [quietvoid/dovi_tool#203] [Q&A] [answered]
Hi,
After trying multiple options for hours, I'm unable to figure out if there is something wrong at my end.

Hardware: MBP 14" M1 Pro with 10 CPU cores + 16 GPU cores

Downloaded the most recent release for macOS: dovi_tool-1.6.0-universal-macOS
Extracted the file (which is unix executable) and placed it in /usr/library/bin

File I'm trying to convert is a HEVC in MKV container

HDR Format: Dolby Vision, Version 1.0, dvhe.07.06, BL+EL+RPU, Blu-ray compatible / SMPTE ST 2086, HDR10 compatible

When I'm running this comman in Terminal:
`ffmpeg -i input.mkv -c:v copy -vbsf hevc_mp4toannexb -f hevc - | dovi_tool -m 2 convert --discard -`

It outputs BL_RPU.hevc file (since EL is discarded in the above step), and when I check the newly generated file in Mediainfo, the Dolby Vision doesn't simply exist. I still tried remuxing is using tsMuxer using BL_RPU.hevc as the video stream and disabling the original video stream, the result is the same (because it is simply remuxing) and the TV (LG C2) also just shows normal HDR pop-up.

What can I be doing wrong?

**Comment by hyllm** [comment, marked as answer]
mediainfo is not showing DoVi data from HEVC file... mux it into MKV and you'll see. :)

**Reply by Shikariiii** [reply]
Kudos! You were right. Since I'm fairly new at this so wasn't aware about it.

I used MKVToolNix to mux into mkv and now the mediainfo shows the correct profile.

However I would like ffmpeg to mux the HEVC to MKV/MP4 container but I guess I'll have to ask around on ffmpeg forums.
